src/plugins/tabs/components/tabs.vue

```diff
@@ -28,19 +28,12 @@
             }"
             @click="showTab(tab, index)"
         >
-            <div class="c-object-label"
-                 :class="{'is-missing': tab.domainObject.status === 'missing'}"
-            >
-                <div class="c-object-label__type-icon"
-                     :class="tab.type.definition.cssClass"
-                >
-                    <span class="is-missing__indicator"
-                          title="This item is missing"
-                    ></span>
-                </div>
-                <span class="c-button__label c-object-label__name">{{ tab.domainObject.name }}</span>
-            </div>
-        </button>
+            <span class="c-button__label c-tabs-view__tab__label">{{ tab.domainObject.name }}</span>
+            <button v-if="openmct.editor.isEditing()"
```

Comment:
since we are getting a prop called isEditing, we can just use that here, instead of checking the editor

Reply:
Got it. Thanks for the heads-up.